### PR TITLE
Lesson 4 / Ivan Ivanov

### PR DESCRIPTION
**Pull request by dmitryrakovets**
I have completed all the tasks, please review the code.
Waiting for feedback :)

### CODE REVIEW

**Comment by rakovets** [comment]
Good. But there are comments on tasks 3 and 6:
- fix ...

File: README.md [hunk 1]
```diff
@@ -10,4 +10,5 @@ Repository for the course "HTML coding for kids"
 2. [Структура HTML-документа](https://github.com/rakavets/html-coding/blob/chapter2/chapter2/technical-task.md)
 3. [Разметка текста](https://github.com/rakavets/html-coding/blob/chapter3/chapter3/technical-task.md)
 4. [Cсылки и изображения](https://github.com/rakavets/html-coding/blob/chapter4/chapter4/technical-task.md)
-5. [Введение в таблицы](https://github.com/rakavets/html-coding/blob/chapter5/chapter5/technical-task.md)
\ No newline at end of file
+5. [Введение в таблицы](https://github.com/rakavets/html-coding/blob/chapter5/chapter5/technical-task.md)
+
```
Comment (rakovets) [inline README.md:14]: Delete empty string because it does not comply with the code convention

File: chapter4/technical-task.md [hunk 2]
```diff
@@ -18,6 +18,8 @@
 13. [Создаем личный vk.com](#statement-of-work-13)
 
 
+
```
Comment (rakovets) [inline chapter4/technical-task.md:21]: and there...

File: technical-task-template.md [hunk 3]
```diff
@@ -23,4 +23,5 @@
 Теоритический материал для выполнения задания 2
 ### Task
 - Задание 2.1
-- ...
\ No newline at end of file
+- ...
+
```
Comment (rakovets) [inline technical-task-template.md:27]: and there...

File: chapter4/task5.html [hunk 4]
```diff
@@ -7,5 +7,7 @@
     <body>
         <h1>Скачиваем файлы</h1>
         <p><a href="">Скачать простой текстовый файл</a></p>
+        <p><a href="">Скачать простой текстовый файл</a></p>
```
Comment (rakovets) [inline chapter4/task5.html:10]: Delete it because copy-past

File: chapter4/task5.html [hunk 5]
```diff
@@ -1,4 +1,4 @@
-!DOCTYPE html>
+<!DOCTYPE html>
```
Comment (rakovets) [inline chapter4/task5.html:1]: Good job!)

File: README.md (hunk 1 above)
Comment (dmitryrakovets) [reply README.md:14]: Okay

File: chapter4/task5.html (hunk 5 above)
Comment (dmitryrakovets) [reply chapter4/task5.html:1]: Thank!)

**Pull request closed without merging**